api.md

```diff
@@ -3406,138 +3312,143 @@ SELECT pg_size_pretty(after_compression_total_bytes) as total
 total | 48 kB
 
 ```
-## hypertable_relation_size() [](hypertable_relation_size)
+---
+
+## hypertable_detailed_size() :community_function: [](hypertable_detailed_size)
```

Comment:
Isn't this in Apache-2 codebase, not TSL?  If so, it shoudn't be marked :community_function:

(Same comment throughout)

Reply:
@mfreed The size functions and some of the new utility functions make calls to tsl code in case the table is a distributed table. But this is not the case for regular hypertables.
So it would work for regular hypertable with Apache. But the function could call tsl code in some circumstances. So is this Apache or Community?

Reply:
you can make the function apache and still have the call to tsl code

Reply:
i mean have the call be conditional and run tsl code only in community edition we have a couple of places that do that already like telemetry that gets additional information

Reply:
This calls tsl code for distributed hypertables. Since we cannot have distributed hypertables in apache, is this good enough?
```
        CASE WHEN is_distributed THEN
            RETURN QUERY SELECT * FROM _timescaledb_internal.hypertable_remote_size(schema_name, table_name);
        ELSE
            RETURN QUERY SELECT *, NULL::name FROM _timescaledb_internal.hypertable_local_size(schema_name, table_name);
        END CASE;
```
The remote_size function invokes the tsl code.

Reply:
yes i think this is fine

Reply:
I believe this should then be updated to remove the :community_function: label?

Reply:
removed community_function label.